src/osw/types.py

```diff
@@ -0,0 +1,23 @@
+import sys
+from pathlib import Path
+
+from pydantic.v1.types import FilePath as PydanticFilePath
+from pydantic.v1.validators import path_validator  # , path_exists_validator
+
+if sys.version_info < (3, 10):
+    NoneType = type(None)
+else:
+    from types import NoneType  # noqa: F401
+
+
+class FilePath(PydanticFilePath):
```

Comment:
A way to avoid validation errors on not (yet) existing filepaths
@SimonStier: Maybe rename class? Maybe add docstring?